### ISSUE TITLE
Monitoring v2 - cannot install Alert-manager and Monitoring together 

### ISSUE THREAD

**Issue by jiaqiluo** [rancher/rancher#28779]
**What kind of request is this (question/bug/enhancement/feature request):**
bug
rancher: `master-4911f8b116eb845ce72bda40fc449f0a4ce54025-head` HA setup

**Steps to reproduce (least amount of steps as possible):**
- try to install the monitoring v2 via the dashboard UI to the local cluster
- leave all option to the default except setting the `cluster type` to be RKE 

**Result:**
errors in the logs of the helm-operation pod
```
Error: rendered manifests contain a resource that already exists. Unable to continue with install: Secret "alertmanager-rancher-monitoring-alertmanager" in namespace "cattle-monitoring-system" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "rancher-monitoring"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "cattle-monitoring-system"
helm.go:84: [debug] Secret "alertmanager-rancher-monitoring-alertmanager" in namespace "cattle-monitoring-system" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "rancher-monitoring"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "cattle-monitoring-system"
rendered manifests contain a resource that already exists. Unable to continue with install
```

**Comment by maggieliu** [comment]
After discussion in slack, it was decided we will do a pre-install hook to handle the secret in the chart. This also means we need to revert the UI change in: https://github.com/rancher/dashboard/issues/1088

**Comment by jiaqiluo** [comment]
The bug fix is validated on rancher:`master-ad697207cadffc75db53b460c644967f58f02cf4-head` HA setup 

 Alert-manager and Monitoring can be installed together into either the local cluster or the downstream cluster successfully
